addons/account/migrations/12.0.1.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,387 @@
+account      / account.tax.template     / type_tax_use (selection)      : selection_keys is now '['adjustment', 'none', 'purchase', 'sale']' ('['none', 'purchase', 'sale']')
+# DONE: post-migration: added the boolean field as a new key in the selection
+
+account      / digest.digest            / kpi_account_total_revenue (boolean): NEW
+# NOTHING TO DO
+
+account      / res.company              / account_bank_reconciliation_start (date): NEW
+account      / res.company              / account_dashboard_onboarding_state (selection): NEW selection_keys: ['closed', 'done', 'just_done', 'not_done']
+account      / res.company              / account_invoice_onboarding_state (selection): NEW selection_keys: ['closed', 'done', 'just_done', 'not_done']
+account      / res.company              / account_onboarding_invoice_layout_state (selection): NEW selection_keys: ['done', 'just_done', 'not_done']
+account      / res.company              / account_onboarding_sale_tax_state (selection): NEW selection_keys: ['done', 'just_done', 'not_done']
+account      / res.company              / account_onboarding_sample_invoice_state (selection): NEW selection_keys: ['done', 'just_done', 'not_done']
+# NOTHING TO DO
+
+account      / res.company              / account_purchase_tax_id (many2one): NEW relation: account.tax
+account      / res.company              / account_sale_tax_id (many2one): NEW relation: account.tax
+??
+
+account      / res.company              / account_setup_bank_data_done (boolean): DEL
+account      / res.company              / account_setup_bank_data_state (selection): NEW selection_keys: ['done', 'just_done', 'not_done']
+account      / res.company              / account_setup_bar_closed (boolean): DEL 
+account      / res.company              / account_setup_coa_done (boolean): DEL 
+account      / res.company              / account_setup_coa_state (selection): NEW selection_keys: ['done', 'just_done', 'not_done']
+account      / res.company              / account_setup_company_data_done (boolean): DEL 
+account      / res.company              / account_setup_fy_data_done (boolean): DEL 
+account      / res.company              / account_setup_fy_data_state (selection): NEW selection_keys: ['done', 'just_done', 'not_done']
+# DONE: post-migration: filled the new *_state fields
+
+account      / res.company              / accounts_code_digits (integer): DEL
+account      / res.company              / incoterm_id (many2one)        : previously in module l10n_be_intrastat
+account      / res.company              / invoice_is_email (boolean)    : NEW 
+account      / res.company              / invoice_is_print (boolean)    : NEW 
+account      / res.company              / invoice_reference_type (selection): NEW selection_keys: function
+account      / res.company              / qr_code (boolean)             : NEW
+account      / res.company              / transfer_account_code_prefix (char): NEW 
```

Comment:
You should copy this prefix from the related chart template (but this doesn't serve for anything for now)